# dji_gimbal config
track_point: true
camera_info_topic: /dji_camera/camera_info
track_point_topic: /ar_marker

yaw_axis: 0
pitch_axis: 4
roll_axis: 3
reset_angle_btn: 0
face_down_btn: 2
toggle_track_btn: 3

Kp: .002

# dji_camera config
camera_info_url: package://dji_gimbal_cam/cfg/zenmuse_z3.yaml
camera_name: zenmuseZ3
camera_frame_id: dji_camera

is_mono: true
transfer: false
